usr.sbin/bhyvectl/bhyvectl.c

```diff
@@ -1753,6 +1757,50 @@ send_start_suspend(struct vmctx *ctx, const char *suspend_file)
 
 	return (send_checkpoint_op_req(ctx, &op));
 }
+
+static int
+send_start_migrate(struct vmctx *ctx, const char *migrate_vm)
+{
+	struct migrate_req req;
+	struct checkpoint_op op;
+	char *hostname, *pos;
+	int rc;
+
+	memset(req.host, 0, MAX_HOSTNAME_LEN);
+	memset(op.host, 0, MAX_HOSTNAME_LEN);
+	hostname = strdup(migrate_vm);
+
+	op.op = START_MIGRATE;
+
+	if ((pos = strchr(hostname, ',')) != NULL ) {
+		*pos = '\0';
+		strncpy(req.host, hostname, MAX_HOSTNAME_LEN);
+		strncpy(op.host, hostname, MAX_HOSTNAME_LEN);
+		pos = pos + 1;
+
+		rc = sscanf(pos, "%d", &(req.port));
+		op.port = req.port;
+
+		if (rc == 0) {
+			fprintf(stderr, "Could not parse the port\r\n");
+			free(hostname);
+			return -1;
+		}
+	} else {
+		strncpy(req.host, hostname, MAX_HOSTNAME_LEN);
+		strncpy(op.host, hostname, MAX_HOSTNAME_LEN);
+		op.host[MAX_HOSTNAME_LEN - 1] = 0;
+
+		/* If only one variable could be read, it should be the host */
+		req.port = DEFAULT_MIGRATION_PORT;
+		op.port = DEFAULT_MIGRATION_PORT;
+	}
```

Comment:
Similar to the `receive_vm_migration` function. Could probably use a similar parsing.